scripts/publish-jars

```diff
@@ -10,27 +10,43 @@ function usage() {
     echo -n \
 "Usage: $(basename "$0" JAR_NAME)
 
-Publish jars to S3 for easy access
+Publish jars to Bintray
 "
 }
 
 
-if [ -d "./scripts/cibuild.d" ]; then
-    for file in ./scripts/cibuild.d/*.sh; do
-        source "${file}"
-    done
-fi
-
-if [ -d "./scripts/cipublish.d" ]; then
-    for file in ./scripts/cipublish.d/*.sh; do
-        source "${file}"
-    done
-fi
-
 if [ "${BASH_SOURCE[0]}" = "${0}" ]; then
     if [ "${1:-}" = "--help" ]; then
         usage
     fi
-    build_ingest_jar
-    upload_ingest_jar "${RF_HOST_USER}"
+    
+    GIT_BRANCH=$(git rev-parse --abbrev-ref HEAD)
+    if [[ "$GIT_BRANCH" != "master" ]]; then
```

Comment:
I don't think `Jenkinsfile.release` runs on a branch necessarily, usually this is run against a specific commit that is labeled with a tag.